tests/core/object/test_class_db.h

```diff
@@ -406,6 +438,14 @@ void validate_argument(const Context &p_context, const ExposedClass &p_class, co
 		}
 
 		TEST_COND(!arg_defval_assignable_to_type, err_msg);
+
+		bool arg_defval_valid_data = arg_default_value_is_valid_data(p_arg.defval, &type_error_msg);
+
+		if (!type_error_msg.is_empty()) {
+			err_msg += " " + type_error_msg;
+		}
+
+		TEST_COND(!arg_defval_valid_data, err_msg);
```

Comment:
Not relevant for this PR, but at some point we need to discuss whether we want to keep these macro aliases or not. I think they are exclusively used for the ClassDB tests - everywhere else the doctest ones are used directly.